pkg/machinery/config/configpatcher/apply_test.go

```diff
@@ -54,7 +61,72 @@ func TestApply(t *testing.T) {
 			bytes, err := out.Bytes()
 			require.NoError(t, err)
 
-			assert.Equal(t, expected, bytes)
+			assert.Equal(t, string(expected), string(bytes))
```

Comment:
assert equal doesn't work with bytes?

Reply:
it does work, but the diff shown is for bytes, which is very hard to read (it does a hexdump diff), while our data is text, so if I cast to string, the diff is much more sane